test-files/cmd/slow/DiffConflateCmdTest/output.osc

```diff
@@ -0,0 +1,662 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<osmChange version="0.6" generator="hootenanny">
+    <create>
+        <node id="-1" version="0" lat="8.6548851891999998" lon="28.1463268213400006" timestamp="">
```

Comment:
While working on API writes I've seen that the `timestamp=""` attribute never contains a valid DTG.  I think that this is something that we need to look into.  It is something that predates your changes so these are good to go but I think that we need to address this sooner than later.  I assume that empty DTG means current insert time in the database but I don't know if that is the desired result.  Looking at your source data there is no `timestamp` attribute so maybe this is correct.  I'll have to try some data that does have a `timestamp` value and see what comes of it.  @bwitham ideas?

Reply:
If I remember correctly, the server has the responsibility for setting the timestamps and not the client.  That should be verified, though.

Reply:
That makes sense.  Thanks.